src/app/shared/formly/extensions/autocomplete.extension.ts

```diff
@@ -0,0 +1,16 @@
+import { FormlyExtension } from '@ngx-formly/core';
+
+/**
+ * Extension that ensures the validators property is always filled.
+ * This prevents some errors that come with undefined properties.
+ */
+export const autocompleteExtension: FormlyExtension = {
+  prePopulate(field): void {
+    const to = field.templateOptions;
+    if (!to || !to.autocomplete) {
+      return;
+    }
+
+    field.templateOptions.attributes = { autocomplete: to.autocomplete };
```

Comment:
This should probably be a spreading operation so that other attributes are not overriden. Only the autocomplete should be set.